src/app/[locale]/layout.tsx

```diff
@@ -24,8 +26,15 @@ export default function RootLayout({
 }: Readonly<Props>) {
   return (
     <html lang={locale}>
-      <body className={inter.className}>
-        <LanguageProvider>{children}</LanguageProvider>
+      <body
+        className={twMerge(
```

Comment:
I think, shadcn added `cn` function to utils folder. It uses `twmerge` under the hood. Please use it